CosmosDb.GraphAPI.Recommender/GraphDBHelper.cs

```diff
@@ -0,0 +1,202 @@
+﻿using CosmosDb.GraphAPI.Recommender.Data.Entites;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.Documents.Client;
+using Microsoft.Azure.Graphs.BulkImport;
+using Microsoft.Azure.Graphs.Elements;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CosmosDb.GraphAPI.Recommender
+{
+    public static class GraphDBHelper
+    {
+        public static DocumentClient CreateClient(string documentServerEndPoint, string authKey)
+        {
+            return new DocumentClient(
+                serviceEndpoint: new Uri(documentServerEndPoint),
+                authKeyOrResourceToken: authKey,
+                connectionPolicy: new ConnectionPolicy
+                {
+                    ConnectionMode = ConnectionMode.Direct,
+                    ConnectionProtocol = Protocol.Tcp
+                });
+        }
+
+        // Database
+
+        public static async Task<Database> CreateDatabaseAsync(DocumentClient documentClient, string databaseId)
```

Comment:
If the only statement in a method as "await Call" then let's do not mark up this method as "async" and just return the method call.